misc/slack-notification/slack.sh

```diff
@@ -1,20 +1,19 @@
 #!/usr/bin/env bash
 
-MESSAGE="Successfully finished running '${ENV0_DEPLOYMENT_TYPE}' deployment on"
-
 if [[ -z "${ENV0_DEPLOYMENT_REVISION}" ]]; then
-  MESSAGE="${MESSAGE} repository's default branch.\n"
+  REVISION="repository's default branch"
 else
-  MESSAGE="${MESSAGE} revision '${ENV0_DEPLOYMENT_REVISION}'.\n"
+  REVISION=$ENV0_DEPLOYMENT_REVISION
 fi
 
-if [[ -z "${ENV0_REVIEWER_NAME}" ]]; then
-  MESSAGE="${MESSAGE} Automatically approved."
-else
-  MESSAGE="${MESSAGE} Approved by: ${ENV0_REVIEWER_NAME}(${ENV0_REVIEWER_EMAIL})"
-fi
+DEPLOYMENT_DETAILS="
+Environment Name: \`${ENV0_ENVIRONMENT_NAME}\`
+Revision: \`${REVISION}\`
+<https://app.env0.com/p/${ENV0_PROJECT_ID}/environments/${ENV0_ENVIRONMENT_ID}/deployments/${ENV0_DEPLOYMENT_LOG_ID}|View deployment>"
```

Comment:
providing a link is really nice.
only problem is the start of the url - for our customers it will be great but for internal use it will not work since it does not start with `dev.dev`